emukit/core/loop/outer_loop.py

```diff
@@ -27,15 +27,15 @@ class OuterLoop(object):
        criterion is met.
     2. Emukit only calculates the next points(s) to try and you evaluate your function or perform the experiment.
     """
-    def __init__(self, candidate_point_calculator: CandidatePointCalculator, model_updater: ModelUpdater,
+    def __init__(self, candidate_point_calculator: CandidatePointCalculator, model_updaters: List[ModelUpdater],
                  loop_state: LoopState = None) -> None:
         """
         :param candidate_point_calculator: Finds next points to evaluate by optimizing the acquisition function
-        :param model_updater: Updates the data in the model and the model hyper-parameters when we observe new data
+        :param model_updaters: Updates the data in the model and the model hyper-parameters when we observe new data
```

Comment:
Isn't it better to call this `models_updater`?

Reply:
I think model updaters is fine because it is two separate model updater objects.

Reply:
then let's change the tex after the "Update_ the data in the model_s..."